eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -423,21 +424,26 @@ object Helpers {
       }
     }
 
-    def firstClosingFee(commitments: Commitments, localScriptPubkey: ByteVector, remoteScriptPubkey: ByteVector)(implicit log: LoggingAdapter): Satoshi = {
+    def firstClosingFee(commitments: Commitments, localScriptPubkey: ByteVector, remoteScriptPubkey: ByteVector, feeratePerKw: Long)(implicit log: LoggingAdapter): Satoshi = {
```

Comment:
I don't see the point of splitting this method in two.

Reply:
This was done for testability and used in [ClosingStateSpec](https://github.com/ACINQ/eclair/pull/1083/files#diff-0d859eeb997ad6e4086f3c413770423fR158), i can change the test to use a hardcoded value, it will be harder to read in the future though.